src/neptune/management/internal/api.py

```diff
@@ -1033,3 +1034,47 @@ def clear_trash(
 
     for error in response.result.errors:
         logger.warning(error)
+
+
+def get_workspace_status(workspace: str, *, api_token: Optional[str] = None) -> Dict[str, int]:
+    """Gets the status of a Neptune workspace including the storage usage and limits.
```

Comment:
It's not a constant set of limits etc. as it depends on your pricing plan etc. so I'm not sure about that.

Reply:
When it tested it, though, all stats are returned even if the plan does not support active/archived projects. (In that case, I get `'activeProjectsLimit': None`). IMO it might be better to explain this than be intentionally vague 😉 Or did you mean something else?

Reply:
`None` means that it's unlimited. It's nothing about whether the user has this particular feature. Historically we have been changing this endpoint from time to time. Especially, there are still things like monitoring hours available (not exposed in Python). So I mean that the result of this function may change at any moment without any new deployment of Python client etc. so don't want to write something on the wall.

Reply:
Let's add these to the description but again, this may be out-of-date quite soon.